Airports/EGMC/Ownership.txt

```diff
@@ -16,4 +16,41 @@ ALTOWNER:Observing London FIR:O:LM:MCT:MCR:THA:TCSE:TCS:TC:LD:LS:LWS:LSC:L
 ALTOWNER:Mentoring TWR/APP:MCTM:MCT:MCRM:MCR:THAM:THA:TCSE:TCS:TC:LD:LS:LWS:LSC:L
 BORDER:MCTWR
 DEPAPT:EGMC
+ARRAPT:EGMC
+
+SECTOR:MCAPP_W:0:3500
```

Comment:
EGMC APP ownership was in EGLC file because LC comes before MC. The EGMC APP sectors are smaller than the Thames ones, and also contained within the Thames sectors, hence they need to appear first in the ESE